File: 3379-score-of-a-string/3379-score-of-a-string.go
func scoreOfString(s string) int {
    sum := 0
    for i := 1; i < len(s); i++ {
        sum += Abs(int(s[i-1]), int(s[i]))
    }
    return sum
}

func Abs(a, b int) int {

    if a-b < 0 {
        return b-a
    }
    return a-b
}

File: 3379-score-of-a-string/3379-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int sum = 0;
        
        for (int i = 1; i < s.length(); i++) {
            sum += Abs(s.charAt(i-1), s.charAt(i)); 
        }

        return sum;
    }

    int Abs(int a, int b){

        if (a > b) {
            return a-b;
        }

        return b-a;
    }
}